zen/client.go

```diff
@@ -449,7 +463,7 @@ func (b *Client) parseBlockData(block *Block) (*types.Block, error) {
 // getHashFromIndex performs the `getblockhash` JSON-RPC request for the specified
 // block index, and returns the hash.
 // https://bitcoin.org/en/developer-reference#getblockhash
-func (b *Client) getHashFromIndex(
+func (b *Client) GetHashFromIndex(
```

Comment:
In parseTransactions exclude coinbase miner reward transaction that are not spendable with transparent txs